Gemfile

```diff
@@ -12,3 +12,4 @@ gem 'rubocop-minitest'
 gem 'rubocop-performance'
 
 gem 'octokit'
+gem 'elasticsearch', '6.8.3'
```

Comment:
This shouldn't go in here. It should be included in any projects that use it (same as the AWS gem), else we end up installing elasticsearch client and it's deps everywhere.

Reply:
You want to add it here: https://github.com/exercism/config/blob/main/exercism_config.gemspec#L43

Reply:
Makes sense. Updated